code/rachel/HTML/lab01.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Salem_the_Bio</title>
+        <link rel="stylesheet" href="https://necolas.github.io/normalize.css/8.0.1/normalize.css">
+        <link rel="stylesheet" href="/Users/rcleav/Desktop/PDX_projects/class_swordfish/code/rachel/HTML/lab01_style.css"> <!--tried using relative path here - didn't work-->
+    </head>    
+    <body>
+        <h1>Salem: the Bio</h1>
+        <p class="intro">Salem is a little American Shorthair cat. He was born in a barn in Idaho, where he was found by a curious pair of cousins who whisked him away to his new home in Virginia. Salem prides himself on his mousing and baby bunny torturing abilities, and if one is lucky enough to garner his love and attention, they will be proudly presented with a loving gift: a freshly captured rodent. Salem is very vocal and will gladly carry on a conversation with anyone willing to speak cat, or even unwilling if food is involved.</p>
+
+        <a href="https://en.wikipedia.org/wiki/American_Shorthair">Click here to learn more about American Shorthairs</a>
+        </p>
+
+        <h2>Salem as a young kitten:</h2>
+        <img src="/Users/rcleav/Desktop/PDX_projects/class_swordfish/code/rachel/HTML/Young_Salem.jpeg" alt="Gray and White Kitten"/>
```

Comment:
* Need to use a relative path here. It's important to use relative paths so the code works everywhere. See me or Merritt if you want help figuring out how to set it.